features/devices/isis/test_isis_route_install.py

```diff
@@ -0,0 +1,244 @@
+import logging as log
+import pytest
+from helpers.otg import otg
+
```

Comment:
comments similar to go test applies here as well

Reply:
updated